### DISCUSSION TITLE
changeset add: accept input from command line options?

### DISCUSSION THREAD

**Discussion by fauxbytes** [changesets/changesets#1322] [Q&A]
Hi. Wondering if at any point it was considered to add command line options to `changeset add`. So that an invocation of the cli e.g.:
```
$ npx changeset --minor --summary "foo"
```
would avoid user interaction and just generate the changeset markdown?
```
$ cat .changeset/itchy-dancers-refuse.md
---
"@the/pkg": minor
---

foo
```

The point-of-which would be to make adding changesets faster.

Thanks,

**Comment by adbutterfield** [comment]
I'd also like to have this feature. I'm setting up a new monorepo, first time using changesets for me. I thought it would be cool to have an github action that creates the changeset file for you, like by adding a tag, and using the PR description as the summary.

**Reply by jslott2sigma** [reply]
I would also like this feature, I see it as:
1. Add argument to specify major, minor, or patch
2. Add argument to specify which packages or all packages
3. Add argument to provide changelog message
4. (Ultimately) Add argument to grab changelog from git commit messages

I'm guessing the first three are relatively straightforward to implement and would cover many usages. How do we move this forward?

**Reply by Sh031224** [reply]
A pull request similar to this content is open. #1121
